chromium_src/net/proxy_resolution/proxy_resolution_service.cc

```diff
@@ -25,3 +25,19 @@ void SetTorCircuitIsolation(const ProxyConfigWithAnnotation& config,
 
 #include "../../../../net/proxy_resolution/proxy_resolution_service.cc"  // NOLINT
 #undef BRAVE_TRY_TO_COMPLETE_SYNCHRONOUSLY
+
+namespace net {
+void ProxyResolutionService::ResetConfigService(
```

Comment:
with https://github.com/brave/brave-core/pull/2647 along with its unittests test fixed (https://github.com/brave/brave-core/pull/2890), we don't need `ResetConfigService` anymore

Reply:
Great! I'll remove it.